### ISSUE TITLE
UI/UX - differentiate closed restaurants from open ones

### ISSUE THREAD

**Issue by shinr** [jtiala/ruoka#10]
Some way to to differentiate already closed restaurants would be useful.  Hiding them or just rendering using a different color scheme.  In mobile view perhaps straight up hiding/collapsing them or sorting differently (with closed ones at the end of the list)

**Comment by jtiala** [comment]
Good idea, but a bit difficult to implement at the moment. The opening times do not have standard syntax, so some kind of parsing should be done. F.ex. Amica Central Station has some additional information in it's closing time field. Also, all restaurants don't have opening times at all.

This is definitely a nice-to-have feature. This could be done at the same time as #9, since normalizing opening times is needed to make this function. 

This can be also extended to differentiate restaurants which menus couldn't be shown for some reason (menu not found for example)
